### ISSUE TITLE
Send Video To Screen: / Second Display Feature to Present the Video on a Second Display ala SMPlayer

### ISSUE THREAD

**Issue by leirbag28** [zaps166/QMPlay2#816]
Would be great if QMPlay2 could have an extra Menu Item where you can Send/Detach the Video To be sent to an external Display in Full screen with no interference of showing controls while the MAIN Window/Controls stay on the Main display  Like SMPlayer which has this feature. I think QMPlay is much better suited for this as it has more abilities like perfect looping, displaying pictures etc Making this an ideal App for dong presentations LIVE!

**Comment by zaps166** [comment]
#113 ?

**Comment by zaps166** [comment]
It's not yet possible, but you can try this:

- press Shift+L to unlock widgets (or uncheck "Widgets->Lock widgets")
- move video dock by pressing on "Video" title bar to the second screen
- resize undocked Video window using mouse to fill entire screen

To dock Video again, double click on "Video" title bar.

**Comment by leirbag28** [comment]
I have done something similar to this before with QMPLay2 but will try this method you mentioned.Sent from my iPhoneOn Dec 22, 2024, at 4:07 AM, Błażej Szczygieł ***@***.***> wrote:﻿
It's not yet possible, but you can try this:

press Shift+L to unlock widgets (or uncheck "Widgets->Lock widgets")
move video dock by pressing on "Video" title bar to the second screen
resize undocked Video window using mouse to fill entire screen

To dock Video again, double click on "Video" title bar.

**Comment by leirbag28** [comment]
> #113 ?

Yes Exactly! By the way I tried what you suggested...pressing SHIFT+L  (Nice to know this) and drag the window to the second screen...I did this and what happens is once I press F or double click to go full screen, It goes FUll Screen but comes BACK to the MAIN display and removes itself from the second display.  The only way I got it to work was to place the Video Window on the second display and then Stretch the window (inconvenient) onto the full display which you then have to hold ALT while dragging it in order to drag the title bar OUTSIDE of the viewable area. The other NORMAL way is of course to just drag the WHOLE QMPLAY2 app onto the second screen and go Full screen but then you have to control ot via keyboard which is fine for some things but not for searching for other videos to ADD while live or Music or Radio stations etc.
